granulate_utils/exceptions.py

```diff
@@ -31,3 +33,10 @@ def __init__(self, container_id: str) -> None:
 class BadResponseCode(Exception):
     def __init__(self, response_code: int):
         super().__init__(f"Got a bad HTTP response code {response_code}")
+
+
+class MissingExePath(Exception):
+    def __init__(self, process: Process):
+        self.process = process
+        super(MissingExePath, self).__init__(
+            f"No exe path was found for pid {process.pid}, threads: {process.threads()}")
```

Comment:
You don't repr the process object itself..

```suggestion
            f"No exe path was found for {process}, threads: {process.threads()}")
```

This will contain the PID as well